exampleCourse/questions/demo/workspace/jupyterlab/info.json

```diff
@@ -14,6 +14,11 @@
       "rewriteUrl": false,
       "gradedFiles": [
           "Workbook.ipynb"
+      ],
+      "syncIgnore": [
+          ".cache/",
+          ".local/share/jupyter",
+          ".npm/"
```

Comment:
I thought these files would no longer be stored in the home dir after we added the `JUPYTER_RUNTIME_DIR` env var. Is that env var not working? Or did I misunderstand what it does?

Reply:
It looks like `JUPYTER_ALLOW_INSECURE_WRITES` is what actually "fixed" the home dotfile perm errors.

Seems that the new runtime dir `/tmp/runtime` is being used for other stuff:

```
root@fcfbff319012:~# ls -a /tmp/runtime
.  ..  nbserver-14.json  nbserver-14-open.html  notebook_cookie_secret
```

Reply:
I wonder whether we should set the following env vars in the container?
```
XDG_CACHE_HOME=/tmp/cache
XDG_DATA_HOME=/tmp/local/share
NPM_CONFIG_CACHE=/tmp/npm
```
This way they won't trigger file notifications at all.

Reply:
Looks like jupyter doesn't respect XDG (and from what I googled, it's intentional):

```bash
root@20da353fac3e:~# echo $XDG_CACHE_HOME
/tmp/cache
root@20da353fac3e:~# ls -a
.  ..  .cache  .config  .ipynb_checkpoints  .jupyter  Workbook.ipynb
root@20da353fac3e:~# ls /tmp/cache
ls: cannot access '/tmp/cache': No such file or directory
```

And on the jupyter side, I don't think we can set cache dirs either:
https://test-jupyter.readthedocs.io/en/rtd-theme/projects/system.html#jupyter-directories